azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/AADAuthenticationFilterAutoConfiguration.java

```diff
@@ -80,6 +89,12 @@ public ResourceRetriever getJWTResourceRetriever() {
                 aadAuthProps.getJwtSizeLimit());
     }
 
+    @Bean
+    @ConditionalOnMissingBean(JWKSetCache.class)
+    public JWKSetCache getJWKSetCache() {
+        return null;
```

Comment:
This method will always return null. It makes no sense.

Reply:
Well with my understanding that @ConditionalOnMissingBean annotation makes it so this method will only get called when JWKSetCache is missingright